sources/packages/forms/src/form-definitions/educationprogramoffering.json

```diff
@@ -164,59 +316,107 @@
         {
-          "label": "How will this be offered?",
+          "label": "Year of study",
           "labelPosition": "top",
-          "optionsLabelPosition": "right",
+          "widget": "choicesjs",
+          "placeholder": "",
           "description": "",
           "tooltip": "",
           "customClass": "",
           "tabindex": "",
-          "inline": false,
           "hidden": false,
           "hideLabel": false,
+          "uniqueOptions": false,
           "autofocus": false,
           "disabled": false,
-          "tableView": false,
+          "tableView": true,
           "modalEdit": false,
-          "values": [
-            {
-              "label": "Part time",
-              "value": "Part Time",
-              "shortcut": ""
-            },
-            {
-              "label": "Full time",
-              "value": "Full Time",
-              "shortcut": ""
-            }
-          ],
+          "multiple": false,
+          "dataSrc": "values",
+          "data": {
+            "values": [
+              {
+                "label": "Year 1",
+                "value": "1"
+              },
+              {
+                "label": "Year 2",
+                "value": "2"
+              },
+              {
+                "label": "Year 3",
+                "value": "3"
+              },
+              {
+                "label": "Year 4",
+                "value": "4"
+              },
+              {
+                "label": "Year 5",
+                "value": "5"
+              },
+              {
+                "label": "Year 6",
+                "value": "6"
+              },
+              {
+                "label": "Year 7",
+                "value": "7"
+              },
+              {
+                "label": "Year 8",
+                "value": "8"
+              },
+              {
+                "label": "Year 9",
+                "value": "9"
+              }
+            ],
+            "resource": "",
+            "json": "",
+            "url": "",
+            "custom": ""
+          },
           "dataType": "",
+          "idPath": "id",
+          "valueProperty": "",
+          "template": "<span>{{ item.label }}</span>",
+          "refreshOn": "",
+          "refreshOnBlur": "",
+          "clearOnRefresh": false,
+          "searchEnabled": true,
+          "selectThreshold": 0.3,
+          "readOnlyValue": false,
+          "customOptions": {},
+          "useExactSearch": false,
           "persistent": true,
           "protected": false,
           "dbIndex": false,
           "encrypted": false,
-          "redrawOn": "",
           "clearOnHide": true,
           "customDefaultValue": "",
           "calculateValue": "",
           "calculateServer": false,
           "allowCalculateOverride": false,
+          "validateOn": "change",
           "validate": {
             "required": true,
-            "onlyAvailableItems": true,
+            "onlyAvailableItems": false,

```

Comment:
If the list is present on form.io _onlyAvailableItems_ must be true for the proper server validation.

Reply:
onlyAvailableItems is checked